server/src/task/test/task.e2e-spec.ts

```diff
@@ -0,0 +1,170 @@
+import { INestApplication, ValidationPipe } from '@nestjs/common';
+import * as request from 'supertest';
+import * as cookieParser from 'cookie-parser';
+import { TestingModule, Test } from '@nestjs/testing';
+import { AuthGuard } from '../../auth/guard';
+import { authSetTokens, buildDefaultModules } from '../../../test';
+import { TaskModule } from '../task.module';
+import {
+  validCreateTaskField,
+  invalidCreateTaskFields,
+  invalidUpdateTaskFields,
+  validUpdateTaskField,
+} from './data';
+
+describe('TaskController (e2e)', () => {
+  let app: INestApplication;
+
+  let token: string;
+  let csrfToken: string;
+
+  beforeAll(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [...buildDefaultModules(), TaskModule],
+      providers: [AuthGuard],
+    }).compile();
+
+    app = moduleFixture.createNestApplication();
+    app.useGlobalPipes(new ValidationPipe());
+    app.use(cookieParser());
+
+    await app.init();
+    app.setGlobalPrefix('api');
+
+    const tokens = await authSetTokens(app);
+    token = tokens.jwtToken;
+    csrfToken = tokens.csrfToken;
+  });
+
+  afterAll(() => {
+    app.close();
+  });
+
+  // create-task test
+
```

Comment:
No need for this comment because it's already described JEST describe method (obvious)! Could you also delete the empty line in this file?

Reply:
Okay, I deleted them!